pkgs/data/fonts/openmoji/default.nix

```diff
@@ -66,6 +66,10 @@ in stdenv.mkDerivation rec {
   buildPhase = ''
     runHook preBuild
 
+    # this is API change in glob >9
+    substituteInPlace helpers/generate-font-glyphs.js \
+      --replace "require('glob').sync" "require('glob').globSync"
```

Comment:
I think this should be put in `postPatch`. Otherwise LGTM.

Reply:
done